src/main/java/com/microsoft/azure/relay/RelayedHttpListenerResponse.java

```diff
@@ -6,20 +6,20 @@
 import java.util.concurrent.CompletableFuture;
 
 import org.eclipse.jetty.http.HttpStatus;
-import org.eclipse.jetty.io.RuntimeIOException;
 
 public class RelayedHttpListenerResponse {
 	private final Map<String, String> headers;
 	private final RelayedHttpListenerContext context;
 	private boolean disposed;
+	private boolean readonly = false;
```

Comment:
> = false [](start = 25, length = 8)

Does it build/compile/save/whatever without warning if the " = false" is omitted here?

Reply:
The value need to initialized in order to compile

---
In reply to: [277057009](https://github.com/Azure/azure-relay-java/pull/41#discussion_r277057009) [](ancestors = 277057009)

Reply:
I don't understand why this private boolean is different than `disposed`. The coding convention isn't consistent between lines 14 and 15.

---
In reply to: [277067283](https://github.com/Azure/azure-relay-java/pull/41#discussion_r277067283) [](ancestors = 277067283,277057009)

Reply:
Didn't see this comment before I pushed...will change it

---
In reply to: [277072160](https://github.com/Azure/azure-relay-java/pull/41#discussion_r277072160) [](ancestors = 277072160,277067283,277057009)